package.json

```diff
@@ -5,9 +5,9 @@
   "license": "GPL-3.0",
   "scripts": {
     "webpack": "NODE_ENV=production webpack -p --bail",
-    "start": "webpack-dev-server --compress",
-    "start:noc": "webpack-dev-server",
-    "startstage": "API_HOST=http://hchung.westus.cloudapp.azure.com:5000 npm start",
+    "start": "webpack-dashboard -- webpack-dev-server --inline --compress",
```

Comment:
what does --inline though?

Also does it matter if the executable used is webpack-dashboard rather than webpack?